fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ExportOperationTest.java

```diff
@@ -675,6 +675,39 @@ public void testBaseExport() throws Exception {
         }
     }
 
+    @Test(groups = { TEST_GROUP_NAME }, dependsOnMethods = { "testGroup" })
+    public void testBaseExportWithMultipleTypes() throws Exception {
+        if (ON) {
+            Parameters parameters = generateParameters(FORMAT_NDJSON, null, null, null);
+            Entity<Parameters> entity = Entity.entity(parameters, FHIRMediaType.APPLICATION_FHIR_JSON);
+
+            Response response = getWebTarget()
+                                    .path(BASE_VALID_URL)
+                                    .queryParam("_outputFormat", FORMAT_NDJSON)
+                                    .queryParam("_type", "Patient,Coverage")
+                                    .queryParam("_type", "Patient")
```

Comment:
seems like you'd want to add a new type here (something other than Patient or Coverage) to prove its doing a union of the values

Reply:
Yes - I can add one of those as well. I wanted to test to make sure we were  not getting 4x results